lib/internal/crypto/keys.js

```diff
@@ -356,21 +415,107 @@ function isKeyObject(key) {
   return key instanceof KeyObject;
 }
 
+// Our implementation of CryptoKey is a simple wrapper around a KeyObject
+// that adapts it to the standard interface. This implementation also
+// extends the JSTransferable class, allowing the CryptoKey to be cloned
+// to Workers.
+// TODO(@jasnell): Embedder environments like electron may have issues
+// here similar to other things like URL. A chromium provided CryptoKey
+// will not be recognized as a Node.js CryptoKey, and vice versa. It
+// would be fantastic if we could find a way of making those interop.
```

Comment:
If it's possible to determine with JS that an object is *any* of these types, then you can use `SymbolHasInstance` to allow `instanceof NodeCryptoKey` to work, but that wouldn't work in reverse unless the other CryptoKeys cooperated.

Reply:
Perhaps eventually. The key issue with CryptoKey is going to be the internal representation to get the EVP_PKEY handle. We'll have to investigate later